### PR TITLE
[NEW] No pinned message shown on empty pin List

### LINKED ISSUE #895: [IMPROVEMENT] Show empty view on pinned messages screen

### PR DESCRIPTION
**Pull request by divyanshub024**
@RocketChat/android

Closes #895 

This PR will show "No pinned message" on empty List.

device-2018-03-16-043829

### CODE REVIEW

**Comment by aniketsingh03** [comment]
@divyanshub024 looks good, but I feel that this and [#931](https://github.com/RocketChat/Rocket.Chat.Android/pull/931) should be similar in terms of icon size, color and text color, so we need to decide upon a common pattern for them. @rafaelks @leonardoaramaki can you guys tell us what size and color would be appropriate for these cases?

File: app/src/main/java/chat/rocket/android/chatroom/ui/PinnedMessagesFragment.kt [hunk 1]
```diff
@@ -88,9 +88,22 @@ class PinnedMessagesFragment : Fragment(), PinnedMessagesView {
...
+                togglePinView(pinnedMessages.size)
             }
 
             adapter.appendData(pinnedMessages)
         }
     }
+
+    private fun togglePinView(size: Int) {
+     if (size == 0){
+         iv_pin_icon.visibility = View.VISIBLE
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/PinnedMessagesFragment.kt]: Change `iv_pin_icon.visibility = View.VISIBLE` to `iv_pin_icon.setVisible(true)` - as well as for the others.

File: app/src/main/java/chat/rocket/android/chatroom/ui/PinnedMessagesFragment.kt [hunk 2]
```diff
@@ -88,9 +88,22 @@ class PinnedMessagesFragment : Fragment(), PinnedMessagesView {
...
+         tv_pin_title.visibility = View.VISIBLE
+         tv_pin_description.visibility = View.VISIBLE
+     }else{
+         iv_pin_icon.visibility = View.GONE
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/PinnedMessagesFragment.kt]: Change `iv_pin_icon.visibility = View.VISIBLE` to `iv_pin_icon.setVisible(false)` - as well as for the others.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -103,6 +103,8 @@
 
     <!-- Pinned Messages -->
     <string name="title_pinned_messages">Pinned Messages</string>
+    <string name="no_pinned_messages">No pinned Messages</string>
+    <string name="no_pinned_description">All the pinned messages   appear here.</string>
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: There are white spaces in this sentence.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -103,6 +103,8 @@
 
     <!-- Pinned Messages -->
     <string name="title_pinned_messages">Pinned Messages</string>
+    <string name="no_pinned_messages">No pinned Messages</string>
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: Please, could you start the `Messages` in lowercase?

Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: Could you add the Brazilian translation for this string?

No pinned Messages -> Nenhuma mensagem pinada.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: Could you add the Brazilian translation for this string?

All the pinned messages appear here -> Todas as mensagens pinadas aparecerão aqui.

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Comment by divyanshub024** [comment]
@filipedelimabrito all changes are made. Pls review it

**Comment by rafaelks** [comment]
@divyanshub024 Did you delete your branch? Cannot find the branch anymore here to update it.

**Comment by divyanshub024** [comment]
@rafaelks sorry! It was a mistake. I'll create a new PR.

**Comment by divyanshub024** [comment]
clone #1066

**Pull request closed without merging**